web/src/OverviewActionBar.tsx

```diff
@@ -224,6 +245,43 @@ export let ButtonRightPill = styled(ButtonRoot)`
   border-radius: 0 4px 4px 0;
 `
 
+const FilterTermTextField = styled(TextField)`
```

Comment:
the right end of the placeholder text is getting cut off in my browser
![image](https://user-images.githubusercontent.com/7453991/121555657-304b5500-c9e1-11eb-8752-a5ca1ad00f5d.png)